iina/SleepPreventer.swift

```diff
@@ -28,11 +64,21 @@ class SleepPreventer: NSObject {
                                               &assertionID)
     if success == kIOReturnSuccess {
```

Comment:
It might be worth converting this to a guard

Reply:
Done.